eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version0/ChannelCodecs0.scala

```diff
@@ -75,7 +75,7 @@ private[channel] object ChannelCodecs0 {
         ("toSelfDelay" | cltvExpiryDelta) ::
         ("maxAcceptedHtlcs" | uint16) ::
         ("isInitiator" | bool) ::
-        ("defaultFinalScriptPubKey" | varsizebinarydata) ::
+        ("defaultFinalScriptPubKey" | optional(provide(true), varsizebinarydata)) ::
```

Comment:
This makes it obvious that we are not changing the existing codec. Also the arg name has been updated:

```suggestion
        ("upfrontShutdownScript_opt" | varsizebinarydata.map(Option(_)).decodeOnly) ::
```